mbed_flasher/erase.py

```diff
@@ -145,8 +145,45 @@ def runner(self, mount_point, filename):
                 self.logger.debug("erase check timed out for %s", mount_point)
                 break
 
+    def erase(self, target_id=None, no_reset=None, method=None):
+        """
+        Erase (mbed) device(s).
+        :param target_id: target_id
+        :param method: method for erase i.e. simple, pyocd or edbg
+        """
+        self.logger.info("Starting erase for given target_id %s", target_id)
+        self.logger.info("method used for reset: %s", method)
+        available_devices = self.get_available_device_mapping()
+
+        if target_id is None:
+            return EXIT_CODE_TARGET_ID_MISSING
+
+        targets_to_erase = self.prepare_target_to_erase(target_id, available_devices)
+
+        if len(targets_to_erase) <= 0:
+            print("Could not map given target_id(s) to available devices")
```

Comment:
Prefer using logger.

Reply:
true - better to change in this phase (originally was prints)